src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -488,6 +478,7 @@ void AbstractNotationPaintView::showContextMenu(const ElementType& elementType,
     }
 
     emit showContextMenuRequested(static_cast<int>(elementType), pos);
+    m_isContextMenuOpen = true;
```

Comment:
If we do this here, shouldn't we also do `m_isPopupOpen = true` in `AbstractNotationPaintView::showElementPopup` for consistency?